### ISSUE TITLE
ApplicationMenu: Incorrect Aria Information

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#4489]
**Describe the bug**

App menu has a few aria issues.

1. The button that functions application menu switcher  doesn't convey the expanded/collapse state info to the screen reader users. The issue occurs as aria-expanded is applied to <div> instead of "Employee" button.
1. Hamburger menu button doesn't provide state and name information to the screen reader users.

**To Reproduce**
**Issue One**
1. Go to https://master-enterprise.demo.design.infor.com/components/applicationmenu/test-personalized-role-switcher-long-title.html
2. Click on 'Some extra...' button
3. Inspect the DOM - The overlay has aria-expanded on the expandable area and not the focused button, so the screen reader wont hear it. Also try to review expandable area.

**Issue Two**
1. Go to https://master-enterprise.demo.design.infor.com/components/applicationmenu/example-index.html
2. Click on 'Hamburger' button
3. Inspect the DOM - Hamburger menu when by default in expanded state and do not inform the screen reader users about the same as it is missing aria-expanded (seperate test).  Also, "Show navigation" is the text within the button that provides the users on how to interact is not clear and is not easily understood as to which actions they need to take to expand/collapse the menu.

**Expected Behavior**
Move the aria and use better off screen labels

**Version**
- ids-enterprise: [e.g. v4.34]

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-32568

**Comment by tmcconechy** [comment]
Timebox this issue

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
